modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/TerraformExecutor.java

```diff
@@ -75,6 +75,16 @@ public SystemCmdResult tfPlan() {
         return executeWithVariables(new StringBuilder("terraform plan -input=false -no-color "));
     }
 
+    /**
+     * Executes terraform plan command and output.
+     *
+     * @return Returns result of SystemCmd executed.
+     */
+    public SystemCmdResult tfPlanWithOutPut() {
```

Comment:
tfPlanWithOutPut -> tfPlanWithOutput (output should be one word.)

Reply:
Renamed.